core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -58,19 +65,64 @@ object ZkUtils extends Logging {
   val IsrChangeNotificationPath = "/isr_change_notification"
   val EntityConfigPath = "/config"
   val EntityConfigChangesPath = "/config/changes"
+  
+  
+  def apply(zkUrl: String, sessionTimeout: Int, connectionTimeout: Int, isZkSecurityEnabled: Boolean): ZkUtils = {
+    val (zkClient, zkConnection) = createZkClientAndConnection(zkUrl, sessionTimeout, connectionTimeout)
+    new ZkUtils(zkClient, zkConnection, isZkSecurityEnabled)
+  }
```

Comment:
It seems that we need to pass in isZkSecurityEnabled to ZKCheckedEphemeral, right?